index.bs

```diff
@@ -320,8 +322,15 @@ Issue: Include cross references to the specs for these hash functions.
     value.  This signals to the listening agent that it should connect to the
     advertising agent to discover updated metadata.
 
-Issue: Add examples of sample mDNS records.
+The advertising agent may add an additional field to the TXT record:
+
+: pw
```

Comment:
Why so terse?  Why not "auth-token"?

Reply:
This is in the TXT record and we need to conserve bytes.  One character keys are possible according to DNS-SD, but two seems reasonable in case we have similar keys.

https://tools.ietf.org/html/rfc6763#section-6.4

Reply:
Oh, right.  I forgot about that.

Well, it's called "-token" every where else now, so "pw" doesn't seem to fit.  Perhaps "tk"?  "at"?